x-pack/test/functional_with_es_ssl/apps/cases/edit_case_form.ts

```diff
@@ -0,0 +1,144 @@
+      await cases.api.createNthRandomCases(1);
+    });
+
+    after(async () => {
+      await cases.api.deleteAllCases();
+    });
+
+    beforeEach(async () => {
+      await common.navigateToApp('cases');
+      await cases.common.goToFirstListedCase();
+      await header.waitUntilLoadingHasFinished();
+    });
+
+    it('edits a case title from the case view page', async () => {
+      const newTitle = `test-${uuid.v4()}`;
+
+      await testSubjects.click('editable-title-edit-icon');
+      await testSubjects.setValue('editable-title-input-field', newTitle);
+      await testSubjects.click('editable-title-submit-btn');
+
+      // wait for backend response
+      await retry.tryForTime(5000, async () => {
+        const title = await find.byCssSelector('[data-test-subj="header-page-title"]');
+        expect(await title.getVisibleText()).equal(newTitle);
+      });
+    });
+
+    it('adds a comment to a case', async () => {
+      const commentArea = await find.byCssSelector(
+        '[data-test-subj="add-comment"] textarea.euiMarkdownEditorTextArea'
+      );
+      await commentArea.focus();
+      await commentArea.type('Test comment from automation');
+      await testSubjects.click('submit-comment');
+
+      // validate user action
+      await find.byCssSelector(
+        '[data-test-subj*="comment-create-action"] [data-test-subj="user-action-markdown"]'
+      );
+    });
+
+    it('adds a tag to a case', async () => {
+      const tag = uuid.v4();
+      await testSubjects.click('tag-list-edit-button');
+      await comboBox.setCustom('comboBoxInput', tag);
+      await testSubjects.click('edit-tags-submit');
+
+      // validate tag was added
+      await testSubjects.existOrFail('tag-' + tag);
+
+      // validate user action
+      await find.byCssSelector('[data-test-subj*="tags-add-action"]');
+    });
+
+    it('changes a case status to in-progress via dropdown menu', async () => {
+      await cases.common.markCaseInProgressViaDropdown();
+      // validate user action
+      await find.byCssSelector(
+        '[data-test-subj*="status-update-action"] [data-test-subj="status-badge-in-progress"]'
+      );
+    });
+
+    it('changes a case status to closed via dropdown-menu', async () => {
+      await cases.common.markCaseClosedViaDropdown();
+
+      // validate user action
+      await find.byCssSelector(
+        '[data-test-subj*="status-update-action"] [data-test-subj="status-badge-closed"]'
+      );
+    });
+
+    it("reopens a case from the 'reopen case' button", async () => {
+      await cases.common.markCaseClosedViaDropdown();
+      await header.waitUntilLoadingHasFinished();
+      await testSubjects.click('case-view-status-action-button');
+      await header.waitUntilLoadingHasFinished();
+
+      await testSubjects.existOrFail('header-page-supplements > status-badge-open', {
+        timeout: 5000,
+      });
+
+      // validate user action
+      await find.byCssSelector(
```

Comment:
Do you mind if we validate the user action?

Reply:
This is the validation of the user action (see comment)